files/build_templates/sonic_debian_extension.j2

```diff
@@ -214,9 +214,11 @@ sudo cp -f $IMAGE_CONFIGS/logrotate/logrotate.d/* $FILESYSTEM_ROOT/etc/logrotate
 sudo cp -f $IMAGE_CONFIGS/systemd/journald.conf $FILESYSTEM_ROOT/etc/systemd/
 
 # Copy interfaces configuration files and templates
+sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config@.service $FILESYSTEM_ROOT/etc/systemd/system/
 sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config.service  $FILESYSTEM_ROOT/etc/systemd/system/
 sudo cp $IMAGE_CONFIGS/interfaces/interfaces-config.sh $FILESYSTEM_ROOT/usr/bin/
 sudo cp $IMAGE_CONFIGS/interfaces/*.j2 $FILESYSTEM_ROOT_USR_SHARE_SONIC_TEMPLATES/
+echo "interfaces-config@.service" | sudo tee -a $GENERATED_SERVICE_FILE
 echo "interfaces-config.service" | sudo tee -a $GENERATED_SERVICE_FILE
```

Comment:
>interfaces-config [](start = 6, length = 17)

Can we remove the old one and use '@' version only?

Reply:
interfaces-config.service is also required to configure the host management and loopback interfaces, so will need both the services.

Reply:
Removed @.service instance as this service is not used to configure loopback IP address.